LangChainQuestionGenerator.py

```diff
@@ -252,21 +257,29 @@ def retrieveLangChainQuestions(config, videoData=None, overwrite=False):
     return questionData
 
 
-def makeRetriever(transcript, embeddings) -> Chroma:
+def makeRetriever(transcript, embeddings, collectionName) -> Chroma:
     """
-    Creates a retriever object using the given transcript and embeddings.
+    Creates a retriever object using the given transcript, embeddings, and collection name.
```

Comment:
I feel `collectionName` here can be mis-understood. Since you are actually passing a Video Caption folder name but in the `makeRetriever` function the naming convention changes to according to how `Chroma` names it. I feel in the docstring could be clarify it on [Line 267](https://github.com/tl-its-umich-edu/annoto-gai/pull/42/commits/78146ae2b0d466f44060ab7769441637f305373a#diff-b8cf3e1e1ff1c0c10e7f4af5206d4b16523ba931c6d402f499b1f5bfa239c446R267)

Reply:
I added some clarification on it in the function description in the latest commit.